Resources/Textures/Decals/medical_decals.rsi/meta.json

```diff
@@ -0,0 +1,88 @@
+{
+  "version": 1,
+  "license": null,
+  "copyright": null,
```

Comment:
Missing contents here.

Reply:
RSI Editor will autofill this if you paste the url of the github link you downloaded the .dmi at.

Reply:
https://discord.com/channels/1168210010233376858/1168210011823026271/1169807706123685968